package/gnss_convertors/gnss_convertors.mk

```diff
@@ -4,9 +4,12 @@
 #
 ################################################################################
 
-GNSS_CONVERTORS_VERSION = v0.3.84
-GNSS_CONVERTORS_SITE = https://github.com/swift-nav/gnss-converters
+# Workaround a Jenkins issue, this is the hash for v0.3.85 but Jenkins
```

Comment:
@schniedergers what is this issue? assuming the solution is to use `git@` instead of `https`, is this still needed?

Reply:
This is the jenkins+buidlroot caching issue, Jenkins permanently caches whatever buildroot downloads, then buildroot will refuse to retry to download if it thinks it has already grabbed the version specified.  In this case we didn't specify the "submodules" option for gnss-converters for version v0.3.85 (and we weren't forcing to use git to download), so it didn't grab the cmake submodule.  The change to use a hash was done just so that buildroot would be forced to download a "new" version with the correct download method (git + submodule fetch).  If we just bump the version from `v0.3.85` to `v0.3.86` we can remove this comment.

Reply:
Also, this comment is already out of date, since @mbavaro pushed a commit on top of `v0.3.85`.

Reply:
makes sense, thanks @silverjam. shouldn't be an issue moving forward now that these have been updated with the submodule directive. @mbavaro can we revert to the previous method (`https`) so that the diff is only the tag updates + submodule directive + header include?